x/intertx/go.mod

```diff
@@ -126,4 +126,6 @@ require (
 
 replace github.com/gogo/protobuf => github.com/regen-network/protobuf v1.3.3-alpha.regen.1
 
-replace github.com/regen-network/regen-ledger/types => ../../types
+replace github.com/regen-network/regen-ledger/api/v2 => ../../api
```

Comment:
```suggestion
```

I don't see `api/v2` anywhere in this g.mod, so I don't think we need the replace directive here.

Reply:
types imports api so we get the following if not included:

```
go: github.com/regen-network/regen-ledger/api/v2@v2.0.0: invalid version: unknown revision api/v2.0.0
```

because it can't find api/v2 (it doesn't exist yet)